ansible/roles/nginx/vars/cchq_ssl.yml

```diff
@@ -20,6 +20,7 @@ nginx_sites:
    location2:
     name: "/static"
     alias: "{{ nginx_static_home }}"
+    add_header: "Access-Control-Allow-Origin *;"
```

Comment:
I don't think this should have the `;` at the end.  causes it to write `add_header Access-Control-Allow-Origin *;;` which makes nginx fail